tests/integration/baseTestAPI.py

```diff
@@ -159,6 +162,21 @@ def setUpClass(cls):
 
             sess.commit()
 
+            # todo: refactor below when we figure out best way for enums in help tables
+            # Create lookup dictionaries
+            js = sess.query(JobStatus)
+            cls.jobStatusDict = {j.name: j.job_status_id for j in js.all()}
+            jt = sess.query(JobType)
+            cls.jobTypeDict = {j.name: j.job_type_id for j in jt.all()}
+            ft = sess.query(FileType)
+            cls.fileTypeDict = {f.name: f.file_type_id for f in ft.all()}
+            fs = sess.query(FileStatus)
+            cls.fileStatusDict = {f.name: f.file_status_id for f in fs.all()}
+            sev = sess.query(RuleSeverity)
+            cls.ruleSeverityDict = {s.name: s.rule_severity_id for s in sev.all()}
+            et = sess.query(ErrorType)
+            cls.errorTypeDict = {e.name: e.error_type_id for e in et.all()}
+
```

Comment:
@jworcestBAH I gave this idea a try over in PR #330. It worked very well for doing the initial db setup scripts. Once that's merged, I'd like to try it out over here and replace this code that dynamically creates lookup dictionaries by querying the lookup tables.